src/Tizen.NUI/src/internal/Interop/InteropLite.Actor.cs

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.Runtime.InteropServices;
+
+namespace Tizen.NUI
+{
+    internal static partial class InteropLite
+    {
+        internal static partial class Actor
+        {
+            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "ActorSetPropertyVector4", CallingConvention = CallingConvention.Cdecl)]
```

Comment:
```suggestion
            [DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Handle_SetPropertyLiteVector4", CallingConvention = CallingConvention.Cdecl)]
```

Could you please add prefix of API something like this, so everyone can know that Input argument is `Dali::Handle`?

Reply:
I decided not to use this bindings. Thank you!